src/utils/format.ts

```diff
@@ -22,12 +22,12 @@ export const decimalFormat = (value: number, decimals = 2): string => {
     return formatter.format(value);
 };
 
-export const convertFahrenheitToCelcius = (temp: number): string => {
-    const formattedCelciusTemp = decimalFormat((temp - 32) * (5 / 9));
-    return formattedCelciusTemp;
+export const convertFahrenheitTosius = (temp: number): string => {
```

Comment:
Typo here, should be I think convertFahrenheitToCelsius

Reply:
This is weird, this didn't comeup when I searched for celcius ... it seems the search was case sensitive... let me correct this...

Reply:
ahh I think I omitted out 'Cel' ... :P